lib/msal-common/src/response/TokenResponse.ts

```diff
@@ -22,6 +22,7 @@ import { AuthResponse } from "./AuthResponse";
 export type TokenResponse = AuthResponse & {
```

Comment:
"state" in response , is it not needed in auth_code?

Reply:
Talked about this offline - state is part of the AuthResponse class, so it will be in the response here.